### ISSUE TITLE
BlueFire config is very weird

### ISSUE THREAD

**Issue by Legendary-Ocean** [ProjectKorra/ProjectKorra#1347]
When values in bluefire config are changed, like cooldown or damage, the outcome in the game is too inconsistent.

The original damage value of FireBlast is 2hp (1 heart) but when bluefire damagefactor is changed to 2, instead of dealing 2 hearts (4 hp)of damage, it deals 4 hearts (9 hp) of damage. When the damage value is changed to 1.5, the damage outcome isnt 1.5 heart/3 hp, its 2.25 hearts of damage.

For the cooldown value, when base FireBlast cooldown is 3000 and the bluefire cooldown factor is 0.5, its supposed to be 1500ms/1.5 sec of cooldown but its 0.7 seconds

> Spigot version: git-Paper-497 (MC: 1.20.4) 
> Core Version: 1.12.0-BETA-12

**Comment by StrangeOne101** [comment]
Are you doing this during the daytime? Because the day factor may also apply

**Comment by Legendary-Ocean** [comment]
Yes, I do it during the day but I made the dayfactor 1. I noticed that Bluefire works fine for core. But for example it doesn't work properly for jedcore. Like FireShots and Fireball, the damage is the same as I said above. The words fireblast in the report threat are wrong, it would be more correct to write fireball. As you said for dayfactor on discord, it should be valid for bluefire as well. JedCore isn't patched for the new attribute system.

**Comment by StrangeOne101** [comment]
Bluefire factor also got changed. So it also needs to be patched in JedCore
